lucida/commandcenter/controllers/Config.py

```diff
@@ -12,41 +12,163 @@
 # models are already saved in `../models`.
 TRAIN_OR_LOAD = 'train' # either 'train' or 'load'
 
+
+
+
+
+#Contains serviceName and data to pass. Needed for batch (and thereby parallel) processing.
+class serviceRequestData(object):
+
+	def __init__(self,nameOfService,argData):
+		self.argumentData = argData
+		self.serviceName = nameOfService
+	
+
+class workFlow(object):
+		def __init__(self):
+			self.currentState = 0; # What state on the state graph
+			self.isEnd = False;
+			self.batchedData = []
+	
+	
+	
+	
+
+class firstWorkflow(workFlow):
+
+	
+	def processCurrentState(self,inputModifierText,inputModifierImage):
+		print "Executing state logic";
+		
+		if(self.currentState==0):
+			print "State 0";
+			self.currentState = 1;
+			self.batchedData = [serviceRequestData("QA",[unicode("How old is Johann")]),serviceRequestData("QA",[unicode("What color is a pug?")])];
+			return;
+		
+		if(self.currentState==1):
+			self.currentState = 2
+			print "State 1";
+			self.batchedData = [serviceRequestData("QA",[unicode("How old is Johann")])]
+			self.isEnd = True
+			return;
+		if(self.currentState==2):
+			self.currentState = 3;
+			print "State 2";
+			self.batchedData = [serviceRequestData("DU",inputModifierText[0])]
+			return;
+						
+		if(self.currentState==3):
+			print "State 3";
+			self.isEnd = True;
+			self.batchedData = [serviceRequestData("QA",inputModifierText[0]),serviceRequestData("DU",inputModifierText[0])]
+			return;
+				
+
+
+
+class QAWF(workFlow):
```

Comment:
from my previous comment, is it smart to have all these stateflows here? these are demos but I don't see this file scaling well as more (complex) stateflows get added.